db/schema.rb

```diff
@@ -70,7 +70,7 @@
     t.string "name", null: false
     t.string "key", null: false
     t.string "contact", null: false
-    t.text "description", limit: 16777215
+    t.text "description", size: :medium
```

Comment:
Have you checked whether all these `limit: 16777215` -> `size: :medium` changes will cause a problem (presume it's OK...)

Reply:
This change is automatic by Rails.  As far as I can see from this Stackoverflow: https://stackoverflow.com/a/19389681/772095 `size: medium` means the same thing as `limit: 16777215`

Reply:
Great :)